doc/policy/packages.md

```diff
@@ -57,3 +57,40 @@ test accepts):
 
    - Warning: Batched fee-bumping may be unsafe for some use cases. Users and application developers
      should take caution if utilizing multi-parent packages.
+
+### Package Fees and Feerate
+
+*Package Feerate* is the total modified fees divided by the total virtual size of all transactions
+in the package after deduplication.
+
+To meet the two feerate requirements of a mempool, i.e., the pre-configured minimum relay feerate
+(`minRelayTxFee`) and dynamic mempool minimum feerate, the total package feerate is used instead of
+the individual feerate. The individual transactions are allowed to be below feerate requirements if
+the package meets the feerate requirements. For example, the parent(s) in the package can have 0
+fees but be paid for by the child.
+
+*Rationale*: This can be thought of as "CPFP within a package," solving the issue of a parent not
+meeting minimum fees on its own. This allows L2 applications to adjust their fees at broadcast time
+instead of overshooting or risking getting stuck/pinned.
+
+*Rationale*: It would be incorrect to use the fees of transactions that are already in the mempool, as
+we do not want a transaction's fees to be double-counted.
+
+### Transactions Are Always Submitted Individually First
+
+Transactions within a package are always validated individually first, and package validation is
+used for the transactions that failed.
+
+*Rationale*: Packages are intended for incentive-compatible fee-bumping. That is, transaction A is a
+"legitimate" fee-bump transaction B only if A is a descendant of B and has a higher feerate.
```

Comment:
nit, most examples seem to designate a parent as A and its child as B (which I like, because that corresponds to time-order, first A, then B), but this seems to be reversed? I believe a few lines below, the convention I'm suggesting is followed, (A is parent, B is child). Also, should "transaction B" be "transaction for B"?